src/simulators/matrix_product_state/matrix_product_state.hpp

```diff
@@ -1089,6 +1055,27 @@ std::vector<reg_t> State::
   return all_samples;
 }
 
+std::vector<reg_t> State::sample_measure_all(const reg_t &qubits, 
+		     uint_t shots, 
+		     RngEngine &rng) {
+  std::vector<reg_t> all_samples;
+  all_samples.resize(shots);
+  std::vector<rvector_t> rnds_list;
+  rnds_list.reserve(shots);
+  for (int_t i = 0; i < shots; ++i) {
+    rvector_t rands;
+    rands.reserve(qubits.size());
+    for (int_t j = 0; j < qubits.size(); ++j)
+      rands.push_back(rng.rand(0., 1.));
+    rnds_list.push_back(rands);
+  }
+  for (int_t i=0; i<static_cast<int_t>(shots);  i++) {
```

Comment:
Changed all the `int_t` to `uint_t`. The reason they were `int_t` was because they were in a parallel section. However, I cancelled the parallelization because it was giving errors.